client/my-sites/plans/ecommerce-trial/upgrade-confirmation/index.tsx

```diff
@@ -22,9 +23,12 @@ const TrialUpgradeConfirmation = () => {
 		wooAdminUrl: selectedSite?.URL ? selectedSite.URL + '/wp-admin/admin.php?page=wc-admin' : '',
 	};
 
+	const currentPlanName = selectedSite?.plan?.product_name_short ?? '';
+
 	return (
 		<>
 			<BodySectionCssClass bodyClass={ [ 'ecommerce-trial-upgraded' ] } />
+			<QuerySitePlans siteId={ selectedSite?.ID ?? 0 } />
 			<Main wideLayout>
 				<PageViewTracker
 					path="/plans/my-plan/trial-upgraded/:site"
```

Comment:
@sixhours, noting for posterity: the `title` prop of `PageViewTracker` should be kept untranslated, as we include that value as-is in the `calypso_page_view` events we record. We want that to stay stable across languages.

Reply:
I suspected there was a reason it was left untranslated, thanks for clarifying!